doc/rest_api/snippet_sharing/use_cases/UC_edit_user.txt

```diff
@@ -0,0 +1,17 @@
+Summary:
+----------------------------
+Title: edit user
+Scope: User Management
+Level: User Goal
+Actors: Authenticated user, Administrator
+Brief: Users can edit their own account information (e.g. change password). Administrators can change all user accounts in the database. Administrators can also change permissions of users.

```

Comment:
What I am missing here (it might be what you understand by "change permissions") some suspend-account functionality to block users which are believed to be spammers (and then to reactivate them if we find out only their account was hacked).

Also some standard maintenance tasks such as reset password seems to be missing?